server/mscalendar/availability.go

```diff
@@ -60,12 +63,21 @@ func (m *mscalendar) syncUsers(userIndex store.UserIndex) (string, error) {
 		return "No connected users found", nil
 	}
 
+	numberOfLogs := 0
 	users := []*store.User{}
 	for _, u := range userIndex {
 		// TODO fetch users from kvstore in batches, and process in batches instead of all at once
 		user, err := m.Store.LoadUser(u.MattermostUserID)
 		if err != nil {
-			return "", err
+			if numberOfLogs < logTruncateLimit {
+				m.Logger.Errorf("Not able to load user %s from user index. err=%v", u.MattermostUserID, err)
```

Comment:
There's not a call to action for the admin to fix this, so we should use `Warn` instead

```suggestion
				m.Logger.Warnf("Not able to load user %s from user index. err=%v", u.MattermostUserID, err)
```